File: SDK/CentralService/java/service/src/main/java/erp/centralservice/service/models/ServiceHeartbeatRequest.java
package erp.centralservice.service.models;

import java.util.LinkedHashMap;
import java.util.Map;

/**
 * 服务心跳请求模型。
 */
public final class ServiceHeartbeatRequest {
    /** 已注册服务实例的唯一标识。 */
    public String id;

    /**
     * 转换为与中心服务接口兼容的 JSON 对象。
     *
     * @return 可直接序列化的键值映射
     */
    public Map<String, Object> toJson() {
        Map<String, Object> m = new LinkedHashMap<String, Object>();
        m.put("id", id);
        return m;
    }
}

File: SDK/CentralService/dotnet/net40/src/CentralService.Service/Models/ServiceHeartbeatRequest.cs
using System.Runtime.Serialization;

namespace CentralService.Service.Models
{
    /// <summary>
    /// 表示发送到 <c>POST /api/Service/heartbeat</c> 的心跳请求体。
    /// </summary>
    [DataContract]
    public sealed class ServiceHeartbeatRequest
    {
        /// <summary>
        /// 获取或设置已注册服务实例的唯一标识。
        /// </summary>
        [DataMember(Name = "id", Order = 1)]
        public string Id { get; set; }
    }
}
